client/lib/simple-payments/utils.js

```diff
@@ -5,3 +5,8 @@ import { SIMPLE_PAYMENTS_PRODUCT_POST_TYPE } from 'lib/simple-payments/constants
 
 export const isValidSimplePaymentsProduct = product =>
 	product.type === SIMPLE_PAYMENTS_PRODUCT_POST_TYPE && product.status === 'publish';
+
+export const decodeProductAttribute = productAttribute =>
+	productAttribute.replace( /(?:&#(\d+);)|(&amp;)/g, ( match, charCode, amp ) =>
```

Comment:
That's strange. Normal posts as well as other custom posts should have similar issues.

I am also wary of RegExp and code conversion due to their popularity in XSS. Question is whether `decodeProductAttribute` can be used as an attack vector?